### PR TITLE
Add `.GenericForm` styles

### PR DESCRIPTION
**Pull request by ohskylab**
TP: https://rockabox.tpondemand.com/entity/8777

See docs for detailed implementation notes.

Using an `Inner` class is a bit awkward here, but necessary until there's a functioning `rb-icon`.

I think this will do as a hook for now, but should be refactored soon.

### CODE REVIEW

**Comment by k0nG** [comment]
On this.

File: src/rb-generic-form/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+define([
+    './rb-generic-form-directive',
+    './rb-generic-form.css'
```
Comment (k0nG) [inline src/rb-generic-form/index.js]: Should include `component/rb-fieldset` here and set it as a dependancy of the module.

Comment (ohskylab) [inline src/rb-generic-form/index.js]: Can that be picked up on the Angular part?

Comment (k0nG) [inline src/rb-generic-form/index.js]: Yeah I guess that's an angular thing. Will let you off.

**Pull request merged**